src/azure-cli/azure/cli/command_modules/iot/tests/hybrid_2020_09_01/_test_utils.py

```diff
@@ -0,0 +1,80 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from os.path import exists
```

Comment:
It would be better to reuse existing file or moving to common folders. avoid duplicate code.